resources/zib/zib-Vaccination-event.xml

```diff
@@ -0,0 +1,158 @@
+  <purpose value="This Immunization resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Vaccination-v4.0(2020EN)](https://zibs.nl/wiki/Vaccination-v4.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-vaccination-v4.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/Vaccination-v4.0(2020EN)" />
+    <name value="zib Vaccination-v4.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Immunization" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Immunization" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Immunization">
+      <path value="Immunization" />
+      <short value="Vaccination" />
+      <comment value="The zib Vaccination is mapped to this Immunization profile and a profile on ImmunizationRecommendation (http://nictiz.nl/fhir/StructureDefinition/zib-Vaccination-request) to align with the intention of FHIR. All past vaccinations are covered using this Immunization profile, while all future vaccinations, including the advised vaccinations, are covered in the ImmunizationRecommendation profile. Both profiles contain the zib mappings, with the exception of the Dose (NL-CM:11.1.4) and Administrator (NL-CM:11.1.6) concepts which are, based on their current definition, not suitable for future vaccinations. Therefore they have only been mapped to the Immunization profile.&#xD;&#xA;&#xD;&#xA;Please note that in contrary to the related zib concepts, `.vaccineCode` and `.occurrence[x]` are mandatory by FHIR and need to be recorded before Immunization resources can be exchanged." />

```

Comment:
> to align with the intention of FHIR

Is dat niet bij elke zib zo? Dat we profilen om de zib met FHIR te alignen.

> while all future vaccinations, including the advised vaccinations, 

Waarom dit specifiek noemen? In de zib definitie gaat het alleen over 'administered' vs 'planned' vaccinations, dus ik zou ook alleen die twee termen gebruiken indien nodig.

>  Both profiles contain the zib mappings, with the exception of the Dose (NL-CM:11.1.4) and Administrator (NL-CM:11.1.6) concepts which are, based on their current definition, not suitable for future vaccinations. Therefore they have only been mapped to the Immunization

Gek dat er in Vaccination-event ingegaan wordt op twee concepten die niet in -request staan.

Dat verwerkend kom ik uit op iets als:

> The zib Vaccination is mapped to this Immunization profile and a profile on ImmunizationRecommendation (http://nictiz.nl/fhir/StructureDefinition/zib-Vaccination-request). All administered vaccinations are covered using this Immunization profile, while all planned vaccinations are covered in the ImmunizationRecommendation profile. This means that zib concepts can exist in both profiles.

> Please note that in contrary to the related zib concepts, `.vaccineCode` and `.occurrence[x]` are mandatory in FHIR and need to be recorded before Immunization resources can be exchanged.

Reply:
Ok! Eens, is bondiger en duidelijker. Ik nam hier als voorbeeld Procedure, daar kan denk ik ook nog wat geschaafd worden.